src/App.tsx

```diff
@@ -1,11 +1,16 @@
 import "./App.css";
 import Dashboard from "./screens/Dashboard/Dashboard";
+import BabyDashboard from "./screens/Dashboard/BabyDashboard/BabyDashboard";
+import CodingDashboard from "./screens/Dashboard/CodingDashboard/CodingDashboard";
+import { Route, Routes } from "react-router-dom";
 
 function App() {
   return (
-    <div className="App">
-      <Dashboard />
-    </div>
+    <Routes>
+      <Route path="/" element={<Dashboard />} />
+      <Route path="/babydashboard" element={<BabyDashboard />} />
```

Comment:
In general make path names kebab case. So `/babydashboard` would become `/baby-dashboard`